.travis.yml

```diff
@@ -31,9 +47,17 @@ jobs:
       os: osx
       if: tag IS blank
       script: travis_wait 30 scripts/ci.bash
-    - name: "Docker build"
+    - name: "Docker build - x86"
+      os: linux
+      dist: trusty
+      arch: amd64
+      env: DOCKER_BUILD="true"
+      if: branch == master AND tag IS blank
+      script: docker build --build-arg githubToken="$GITHUB_TOKEN" --build-arg codeServerVersion="$VERSION" --build-arg vscodeVersion="$VSCODE_VERSION" -t codercom/code-server:"$TAG" -t codercom/code-server:v2 .
+    - name: "Docker build - ARM"
       os: linux
       dist: trusty
+      arch: arm64
```

Comment:
If we're building an arm64 version of the Dockerfile then we'll want to change the tag that automatically gets deployed as well otherwise it'll overwrite the amd64 version. Maybe set the `DOCKER_BUILD` environment variable to the arch or something like that.

Reply:
we can append `-$(uname -m)` on the tag if needed.